testng-eclipse-plugin/src/main/org/testng/eclipse/util/CustomSuite.java

```diff
@@ -228,6 +231,10 @@ private void createXmlFileFromTemplate(XMLStringBuffer suiteBuffer, String fileN
       }
     } catch (IOException e) {

```

Comment:
@nickboldt , I think you can catch the generic `Exception` here rather than add more catch block which fail the travis build